charts/posthog/templates/error-on-both-externalService-and-service-enabled.yaml

```diff
@@ -0,0 +1,5 @@
+{{- if false -}}
+# :TRICKY: This template raises an error if any externalService is configured without service.enabled being false in values.yaml.
+#   It doesn't actually define any real kubernetes resources
+{{- end -}}
+{{- include "snippet.error-on-both-externalService-and-service-enabled" . -}}
```

Comment:
Minor request: please merge this file with `charts/posthog/templates/error-on-removed-values.yaml` to `charts/posthog/templates/error-on-invalid-values.yaml`, merge the tpl file with the other tpl file.

After that this looks great to me. It would be awesome to add some helm-unittest coverage here, but not required immediately.

Reply:
Done. I couldn't add an error test case because of [this bug](https://github.com/quintush/helm-unittest/issues/144), but it should be possible to do so once the bug fix is released.